package.json

```diff
@@ -37,17 +37,19 @@
     "node": ">=10.12.0"
   },
   "dependencies": {
+    "@electron/get": "^1.12.4",
     "@malept/cross-spawn-promise": "^1.1.1",
     "colors": "^1.3.3",
     "debug": "^4.1.1",
     "detect-libc": "^1.0.3",
+    "extract-zip": "^2.0.1",
     "fs-extra": "^9.0.1",
     "got": "^11.7.0",
     "lzma-native": "^7.0.1",
-    "node-abi": "^2.19.2",
-    "node-gyp": "^7.1.0",
+    "node-abi": "^2.30.0",
+    "node-gyp": "^8.0.0",
```

Comment:
FYI, if you're going to be upgrading `node-gyp` here, I'd consider this a breaking change since the external dependencies have changed from Python 2 to Python 3.

Also, the version of Python 3 is too new for the existing CI images, so I would suggest cherry-picking this commit: https://github.com/electron/electron-rebuild/pull/669/commits/6d2f8df335d3112a159550dfe6ae9077aafd871c

Reply:
This was resolved in `electron-rebuild@3.0.0`, so the PR needs to be synced with the default branch.